client/src/components/EditProfile/EditPhoto/EditPhoto.tsx

```diff
@@ -0,0 +1,69 @@
+import { useState } from 'react';
+import CssBaseline from '@material-ui/core/CssBaseline';
+import Paper from '@material-ui/core/Paper';
+import Box from '@material-ui/core/Box';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+import Avatar from '@material-ui/core/Avatar';
+import Typography from '@material-ui/core/Typography';
+import useStyles from './useStyles';
+import DeleteSharpIcon from '@material-ui/icons/DeleteSharp';
+
+export default function EditPhoto(): JSX.Element {
+  const [file, setFile] = useState<string>('');
+
+  const classes = useStyles();
+
+  function importData() {
+    const input: HTMLInputElement = document.createElement('input');
+    input.type = 'file';
+    input.accept = 'image/png, image/jpeg';
+    input.onchange = () => {
+      const files = input.files;
+      setFile(files === null ? '' : URL.createObjectURL(files[0]));
+    };
+    input.click();
+  }
+
+  function removeFile() {
+    setFile('');
+  }
+
+  return (
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
```

Comment:
The `CssBaseline` is a global reset component, we need to include it only once in the App.tsx :)

Reply:
I see. Thanks for your reminder.